docs/_templates/quickstart.rst

```diff
@@ -0,0 +1,84 @@
+.. _quick_start:
+
+Quick Start
+===========
+
+This quick-start contains an overview of how the ``xpdAn`` software works.
+To understand more details about it, please refer to the detailed documentation in :ref:`xpdu`
+
+Please use this page as a reminder of the workflow and to copy & paste code snippets into the
+ipython terminals that are controlling your experiment and analysis.  After
+pasting, hit enter.
+
+Remember, to post questions about anything XPD, including software, and to see archived answers, at the `XPD-Users Google group
+<https://groups.google.com/forum/#!forum/xpd-users;context-place=overview>`_ . If you are not already a member please request to join
+the community
+
+XPD Data Analysis Workflow Summary
+------------------------------------
+
+This is the summary of the steps for analysis data at XPD.
+The steps are explained below.
+Carry out the steps in this order to ensure a successful analysis.
+
+  1. If you haven't already, join `XPD-Users Google group <https://groups.google.com/forum/#!forum/xpd-users;context-place=overview>`_ . Look here for answers if you get stuck and if the answer to your question is not already there, please ask it!
+
+  2. Start the analysis by running ``{{ start_analysis }}`` in a terminal
+
+  3. Your data will be automatically saved and visualized via the analysis pipeline.
+
+    * The data will be saved in the directory
+      ``.../xpdUser/tiff_base/<Sample_name_from_spreadsheet>``.
+
+    * The pipeline will save:
+
+        1. Dark corrected image (as ``.tiff``)
+        2. Mask (as ``.msk`` and ``.npy``,
+           ``.msk`` files are read by ``fit2d`` while ``.npy`` files are read
+           by numpy). The frequency of mask production is controlled by
```

Comment:
how do the users use `mast_settings`?    Can we instead make a sub-bullet something like
1. Mask (...)
    1. by default a mask is auto-computed on the first image of the scan and this mask is used for all subsequent images in the real-time analysis (you are always given an unmasked tiff file so you can change the masking behavior later)
    1. alternative behavior avialable is to auto-compute a new mask for each image or to turn off masking altogether during real-time analysis
    1. to change the behavior, stop the scan and run `mask_settings` at the command line, then restart the scan.